moveit_ros/moveit_servo/include/moveit_servo/servo.h

```diff
@@ -86,20 +86,12 @@ class Servo
   /** \brief Get the parameters used by servo node. */
   const std::shared_ptr<moveit_servo::ServoParameters>& getParameters() const;
 
-  /** \brief Change the controlled link. Often, this is the end effector
-   * This must be a link on the robot since MoveIt tracks the transform (not tf)
-   */
-  void changeRobotLinkCommandFrame(const std::string& new_command_frame)
```

Comment:
Why delete this?

Reply:
I was going to replace this function call with a service (so it could be done through the component interface also).  However, I don't think that change belongs in this PR.  I'll put this back.

Reply:
I'm glad you commented it. This is a useful function that I would like to keep.